crates/meroctl/src/cli/context/alias.rs

```diff
@@ -64,3 +64,58 @@ impl ContextAliasCommand {
         Ok(())
     }
 }
+
+#[derive(Debug, Parser)]
+#[command(name = "use", about = "Set the default context")]
+pub struct UseCommand {
+    pub context_id: Alias<ContextId>,
+}
+
+impl UseCommand {
+    pub async fn run(self, environment: &Environment) -> EyreResult<()> {
+        let config = load_config(&environment.args.home, &environment.args.node_name)?;
+        let multiaddr = fetch_multiaddr(&config)?;
+
+        // Create "default" alias for the specified context ID
+        let default_alias: Alias<ContextId> =
+            "default".parse().expect("'default' is a valid alias name");
+
+        // Resolve the input (whether it's an alias or a context ID)
+        let lookup_response =
+            lookup_alias(multiaddr, &config.identity, self.context_id.clone(), None)
+                .await
+                .wrap_err("Failed to resolve context alias")?;
+
+        // Extract the context ID from the lookup response
+        let resolved_context_id = lookup_response
+            .data
+            .value
+            .ok_or_else(|| eyre::eyre!("Failed to resolve context: no value found"))?;
```

Comment:
we cannot GUARANTEE the provided `context` WILL be an already existing alias.

So we should be calling `resolve_alias()` instead, extracting the `ContextId` with `ResolveResponse::value`